File: seava.mod.ad.ui.extjs/src/main/resources/webapp/seava/ad/ui/extjs/ds/JobTimerLov_Ds.js
/**
 * Solutii Ecommerce, Automatizare, Validare si Analiza | Seava.ro
 * Copyright: 2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
	
Ext.define("seava.ad.ui.extjs.ds.JobTimerLov_Ds", {
	extend: 'Ext.data.Model',
	
	statics: {
		ALIAS: "ad_JobTimerLov_Ds"
	},
	
	
	fields: [
		{name:"jobContextId", type:"string"},
		{name:"jobContext", type:"string"},
		{name:"jobName", type:"string"},
		{name:"id", type:"string"},
		{name:"clientId", type:"string"},
		{name:"name", type:"string"},
		{name:"description", type:"string"},
		{name:"active", type:"boolean"},
		{name:"refid", type:"string"}
	]
});

File: seava.mod.ad.presenter/src/main/java/seava/ad/presenter/impl/scheduler/model/JobTimerLov_Ds.java
/**
 * Solutii Ecommerce, Automatizare, Validare si Analiza | Seava.ro
 * Copyright: 2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package seava.ad.presenter.impl.scheduler.model;

import seava.ad.domain.impl.scheduler.JobTimer;
import seava.j4e.api.annotation.Ds;
import seava.j4e.api.annotation.DsField;
import seava.j4e.api.annotation.SortField;
import seava.j4e.presenter.impl.model.AbstractTypeLov_Ds;

@Ds(entity = JobTimer.class, sort = {@SortField(field = JobTimerLov_Ds.f_name)})
public class JobTimerLov_Ds extends AbstractTypeLov_Ds<JobTimer> {

	public static final String ALIAS = "ad_JobTimerLov_Ds";

	public static final String f_jobContextId = "jobContextId";
	public static final String f_jobContext = "jobContext";
	public static final String f_jobName = "jobName";

	@DsField(join = "left", path = "jobContext.id")
	private String jobContextId;

	@DsField(join = "left", path = "jobContext.name")
	private String jobContext;

	@DsField(join = "left", path = "jobContext.jobName")
	private String jobName;

	public JobTimerLov_Ds() {
		super();
	}

	public JobTimerLov_Ds(JobTimer e) {
		super(e);
	}

	public String getJobContextId() {
		return this.jobContextId;
	}

	public void setJobContextId(String jobContextId) {
		this.jobContextId = jobContextId;
	}

	public String getJobContext() {
		return this.jobContext;
	}

	public void setJobContext(String jobContext) {
		this.jobContext = jobContext;
	}

	public String getJobName() {
		return this.jobName;
	}

	public void setJobName(String jobName) {
		this.jobName = jobName;
	}
}
